helm/values-prod.yaml

```diff
@@ -69,10 +69,15 @@ microservice-chart:
     COSMOS_URI: https://pagopa-p-weu-afm-marketplace-cosmos-account.documents.azure.com:443/
     ISSUER_RANGE_TABLE: 'pagopapweuafmsaissuerrangetable'
     AMOUNT_THRESHOLD: '0'
+    ELASTIC_APM_SERVICE_NAME: pagopa-afm-calculator
+    ELASTIC_APM_APPLICATION_PACKAGES: it.gov.pagopa.afm.calculator
+    ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
+    ELASTIC_APM_LOG_SENDING: 'true'
   envSecret:
     APPLICATIONINSIGHTS_CONNECTION_STRING: ai-p-connection-string
     COSMOS_KEY: afm-marketplace-p-cosmos-pkey
     AFM_SA_CONNECTION_STRING: afm-p-sa-connection-string
+    ELASTIC_APM_SECRET_TOKEN: elastic-apm-secret-token
```

Comment:
@pasqualespica where can i get the tokens for UAT and PROD?

Reply:
you find it in to secret `quickstart-apm-token` of `elastic-system` namespace of each env

Reply:
thank you